src/main/java/se/liu/ida/hefquin/engine/queryproc/impl/optimizer/utils/CardinalitiesCache.java

```diff
@@ -0,0 +1,22 @@
+package se.liu.ida.hefquin.engine.queryproc.impl.optimizer.utils;
+
+import se.liu.ida.hefquin.engine.queryplan.PhysicalPlan;
+
+import java.util.*;
+
+public class CardinalitiesCache {
```

Comment:
This is the latest version of this class, right?

Reply:
Yes